TestClientNotificationStorm is broken on FreeBSD

#### System information

Geth version: `cffdc43954c5f5fa4364ddfbf3881cdfeb667fb7`
OS & Version: FreeBSD

#### Expected behaviour

`TestClientNotificationStorm` passes on FreeBSD

#### Actual behaviour

```
> go run build/ci.go test -coverage ./rpc/
>>> /usr/local/go/bin/go test -ldflags -X main.gitCommit=cffdc43954c5f5fa4364ddfbf3881cdfeb667fb7 -X main.gitDate=20210211 -p 1 -covermode=atomic -cover ./rpc/
--- FAIL: TestClientNotificationStorm (2.40s)
    client_test.go:425: didn't get expected error
FAIL
coverage: 84.8% of statements
FAIL	github.com/ethereum/go-ethereum/rpc	6.768s
FAIL
util.go:47: exit status 1
exit status 1
```

#### Steps to reproduce the behaviour

`go run build/ci.go test -coverage ./rpc/`

--- a/rpc/client_test.go
+++ b/rpc/client_test.go
@@ -427,7 +427,7 @@ func TestClientNotificationStorm(t *testing.T) {
 	}
 
 	doTest(8000, false)
-	doTest(23000, true)
+	doTest(24000, true)
 }
 
 func TestClientSetHeader(t *testing.T) {

